tsql/TSqlParser.g4

```diff
@@ -472,9 +477,44 @@ class_type_for_parallel_dw
     | OBJECT
     ;
 
+//https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-broker-priority-transact-sql
```

Comment:
Please add at least one white space after comment slashes for better readability.